packages/atlas/src/components/_ypp/YppAuthorizationStepModal/YppAuthorizationStepModal.tsx

```diff
@@ -0,0 +1,140 @@
+          primaryButton: {
+            text: 'Authorize with YouTube',
+          },
+          component: <RequirementsDialog />,
+        }
+      case 'details':
+        return {
+          title: 'Details',
+          description: 'We need your email address to send you payment information. No spam or marketing materials.',
+          primaryButton: {
+            text: 'Continue',
+          },
+          component: <DetailsFormDialog />,
+        }
+      case 'terms-and-conditions':
+        return {
+          title: 'Terms & conditions',
+          description:
+            'Once automatic YouTube videos sync is available, in order for it to work, your Atlas channel [NEEDS TO DO WHAT?]. This is purely a technical measure and does not affect ownership and rights to the content uploaded to you Atlas channel.',
+          primaryButton: { text: 'Accept terms & sign' },
+          additionalSubtitleNode: (
+            <AdditionalSubtitle>
+              <Text variant="h400" as="span">
+                Automatic YouTube sync
+              </Text>{' '}
+              <Text variant="t100" as="span" color="colorText">
+                Coming later this year
+              </Text>
+            </AdditionalSubtitle>
+          ),
+          component: <TermsAndConditionsDialog />,
+        }
+      case 'summary':
+        return {
+          title: 'Congratulations!',
+          description: (
+            <>
+              Congratulations! You just received 200 JOY as your Atlas channel is now officially enrolled in the YouTube
+              Partner Program and tied with a YouTube channel.{' '}
+              <Text variant="t200" as="p" margin={{ top: 2 }} color="colorText">
+                All information around your activity in the program can be found in the{' '}
+                <Button variant="primary" _textOnly>
+                  YPP page in Studio
+                </Button>
+                .
+              </Text>
+            </>
+          ),
+          primaryButton: { text: 'Go to dashboard' },
+          component: <Img src={appScreenshot} />,
+        }
+      case 'connect-with-yt':
+        return {
+          title: 'Connect with YouTube?',
+          description:
+            'Reupload and backup your YouTube videos to receive to receive a guaranteed payout in the YouTube Partner Program.',
+          primaryButton: {
+            text: 'Authorize with YouTube',
+          },
+        }
+    }
+  }, [channels, step])
+  return (
+    <DialogModal
+      show={show}
+      dividers
+      primaryButton={authorizationStep?.primaryButton}
+      secondaryButton={{ text: !isSummary ? 'Back' : 'Close', onClick: onBackClick }}
+      additionalActionsNode={
+        isConnectWithYt ? (
+          <Button variant="tertiary">Learn more</Button>
+        ) : (
+          !isSummary && (
+            <Button variant="tertiary" onClick={onCancelClick}>
```

Comment:
On the select channel, the Cancel button shouldn't be present.

Reply:
U sure? https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1637%3A121102

Reply:
Sorry, I meant the `Back` button
![image](https://user-images.githubusercontent.com/51168865/190177258-1169ef54-0dbb-4d76-9947-cf2c4a9f35ac.png)